layouts/PressPage/PressPage.tsx

```diff
@@ -0,0 +1,158 @@
+
+interface ContentPageProps {
+  meta: {
+    title?: string;
+    subtitle?: string;
+    description?: string;
+    noindex?: boolean;
+    articles: PressListItemProps[];
+  };
+  children: React.ReactNode;
+}
+
+export const PressPage = ({
+  meta: { title, description, noindex, articles },
+}: ContentPageProps) => {
+  return (
+    <>
+      <Head title={title} description={description} noIndex={noindex} />
+      <Layout border="none" behaviour="floating">
+        <SectionHeader
+          subtitle="Press"
+          title="Recent publications and media resources"
+          description={
+            <>
+              <Button
+                as="a"
+                href="mailto:press@goteleport.com"
+                shape="outline"
+                variant="secondary"
+              >
+                Request information
+              </Button>
+            </>
+          }
+          bg="wave"
+        />
+        <Section bg="grayGradient">
+          <Centrator py={[3, 11]}>
+            <PressList items={sortItems(articles)} />
+          </Centrator>
+        </Section>
+        <Section bg="grayGradient">
+          <Centrator flexDirection="column" py={[3, 11]}>
+            <StyledHeader>Media Resources</StyledHeader>
+            <StyledCards>
+              <StyledCard>
+                <StyledCardTitle>Contact information </StyledCardTitle>
+                <StyledCardContent>
+                  Please use the contact information below to reach us regarding
+                  publications, blog posts, multi media, and other press related
+                  topics.
+                </StyledCardContent>
+                <Button
+                  as="a"
+                  href="mailto:press@goteleport.com"
+                  variant="secondary"
+                  shape="outline"
+                  width={["100%", "auto"]}
+                >
+                  press@goteleport.com
+                </Button>
+              </StyledCard>
+              <StyledCard>
+                <StyledCardTitle>Press kit</StyledCardTitle>
+                <StyledCardContent>
+                  Our press kit contains official logos, product photos, and
+                  exec team pictures. Please contact if you need additional
+                  resources. Thank you!
+                </StyledCardContent>
+                <Button
+                  as="a"
+                  href="mailto:press@goteleport.com"
+                  variant="secondary"
+                  shape="outline"
+                  width={["100%", "auto"]}
+                >
+                  Request presskit
+                </Button>
+              </StyledCard>
+            </StyledCards>
+          </Centrator>
+        </Section>
+        <OurOffices />
+        <TryTeleport />
+      </Layout>
+      <Footer />
+      <Drift appId={process.env.NEXT_PUBLIC_DRIFT_ID} />
+    </>
+  );
+};
+
+const StyledHeader = styled("h3")(
```

Comment:
```suggestion
const StyledHeader = styled("h2")(
```

Maybe using the `h2` tag would be more correct here?